commands/announce.py

```diff
@@ -207,6 +208,28 @@ def lift_lockdown(self, event, channels):
                         break
         event.msg.reply("Unlock command has successfully completed!")
 
+    @Plugin.command('verification', '<level:str> [reason:str...]')
+    @command_wrapper()
```

Comment:
might wana add `log=False` as param here since you already do the logging, otherwise it'll both log that as well as the default "x used command y"